src/addons/sourcemod/scripting/zr/api/class.api.inc

```diff
@@ -52,6 +52,7 @@ void APIClassInit()
     CreateNative("ZR_GetClassByIdentifier", APIGetClassByIdentifier);
     CreateNative("ZR_GetClassDisplayName", APIGetClassDisplayName);
     CreateNative("ZR_GetClassIdentifier", APIGetClassIdentifier);
+    CreateNative("ZR_SendMenuClass", APISendMenuClass);
```

Comment:
keep the same name as in the zr code to be consitent `ZR_MenuClass` and `APIMenuClassr`